erpnext/stock/doctype/delivery_settings/delivery_settings.json

```
@@ -77,10 +81,34 @@
    "fieldtype": "Link",
    "label": "Default Fulfilment Partner",
    "options": "Fulfillment Partner"
+  },
+  {
+   "fieldname": "delivery_defaults_section",
```

```suggestion
   "fieldname": "delivery_defaults",
```
Why are we changing the fieldname?? The fieldname should be the same while moving the fields

---

Made correction to the field name